.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject/private/
```

Comment:
This looks to be more of a personal gitignore entry -- it belongs in your `.gitignore_global` rather than the project's `.gitignore`